src/login.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Body, Controller, HttpCode, Post } from '@nestjs/common';
+import { StatusCodes } from 'http-status-codes';
```

Comment:
I suggest you to use nestjs `HttpStatus`.

Reply:
Resolved on 1d84f9a3eaf92a35983aebc14843850a8cea13ab, 1d84f9a3eaf92a35983aebc14843850a8cea13ab and 1d84f9a3eaf92a35983aebc14843850a8cea13ab.